doc-src/install.rst

```diff
@@ -56,32 +56,35 @@ example:
 Build time
 ----------
 
-Systems used for building images need the run-time prerequisites, plus:
+Systems used for building images need the run-time dependencies, plus
+something to actually build the images.
 
-* Bash 4.1+
-
-and optionally:
-
-* `Docker <https://www.docker.com/>`_ 17.03+
-* internet access or Docker configured for a local Docker hub
-* root access using :code:`sudo`
-
-Older versions of Docker may work but are untested. We know that 1.7.1 does
-not work.
+A common choice is `Docker <https://www.docker.com/>`_, along with internet
+access or configuration for a local Docker repository. Our wrapper scripts
+for Docker need Docker 17.03+ and expect to run the :code:`docker` command
+under :code:`sudo`. (Older versions of Docker may work but are untested. We
+know that 1.7.1 does not work.)
 
 Test suite
 ----------
 
-In order to run the test suite on a run or build system (you can test each
-mode independently), you also need:
+To run the test suite, you also need:
 
-* Bash 4.1+
-* Python 2.6+
 * `Bats <https://github.com/sstephenson/bats>`_ 0.4.0
-* wget
+* Bash 4.1+, for Bats and to make programming the tests tractable
+* Python 2.7+, for building some of the tests
+* Wget, to download stuff for some of the test images
+
+Image building software tested, with varying levels of thoroughness:
+
+* Shell scripts with various manual bootstrap and :code:`ch-run`
+* Docker
+* `Buildah <https://github.com/containers/buildah>`_
+* `skopeo <https://github.com/containers/skopeo>`_ and
+  `umoci <https://github.com/openSUSE/umoci>`_
 
-Note that without Docker on the build system, some of the test suite will be
-skipped.
+Most of the test images are built with Docker, so the test suite is less
+thorough if it's not installed.
```

Comment:
Is architecture worth mentioning, at least while we have some hard coding (#248)?

Reply:
Yes, that's a good point.